src/WidgetRegistry.ts

```diff
@@ -113,7 +121,7 @@ export class WidgetRegistry extends Evented implements WidgetRegistry {
 		promise.then((widgetCtor) => {
 			this.registry.set(widgetLabel, widgetCtor);
 			this.emit({
-				type: `loaded:${widgetLabel}`
+				type: getEventType(widgetLabel)
```

Comment:
It could be abstracted to a function that accepts the widgetLabel (and the instance if not a method on the class) but it didn't really seem worthwhile to create a function (as this was existing code), but I could if you think it is needed?

```
emitLoadedEvent(widgetLabel: RegistryLabel) {
    this.emit({
        type: getEventType(widgetLabel)
    });
}
```